### PR TITLE
GH-55 Added relase script

### PR DESCRIPTION
**Pull request by whiskeysierra**
See #55

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,22 @@
...
 
-# stop mvn install from running
-install: true
-
-script: mvn -B -V package $( [[ "$TRAVIS_JDK_VERSION" == "oraclejdk8" ]] && echo javadoc:javadoc ) test
+script: ./mvnw clean verify
 
 after_success:
-    - mvn -B -V clean test jacoco:report coveralls:report
+  - ./mvnw coveralls:report
```
Comment (vadeg) [inline .travis.yml]: `jacoco:report` was removed. Is it fine?

Comment (whiskeysierra) [reply .travis.yml]: I changed the `script` to include the `verify` phase which `jacoco:report` binds to by default. Has the positive side effect that as soon as we start to impose coverage rules (soon?! :crossed_fingers:) those violations will break the build already.

**Comment by Jan-M** [comment]
:+1:

File: .travis.yml (hunk 1 above)
Comment (vadeg) [reply .travis.yml]: 👍

**Pull request merged**